src/Cake.Common.Tests/Unit/Build/BitbucketPipelines/Data/BitbucketPipelinesRepositoryInfoTests.cs

```diff
@@ -85,7 +85,7 @@ public void Should_Return_Correct_Value()
                 var result = info.Tag;
 
                 // Then
-                Assert.Equal("BitbucketPiplines", result);
+                Assert.Equal("BitbucketPipelines", result);
```

Comment:
This one is actually a test that either doesn't run or is about to start failing?